communities.json

```diff
@@ -125,5 +125,12 @@
       "inviteCode": "obsproject",
       "githubUrl": "https://github.com/obsproject/obs-studio"
     }
+    {
+      "logo": "rpcs3.svg",
+      "title": "RPCS3",
+      "quote": "RPCS3 is the world's first PlayStation 3 emulator. We use Discord to provide support for our users, has allowed easy collaboration between contributors and is a great place for our patrons to discuss development.",
```

Comment:
read that again, please

Reply:
Maybe reword to something like "*We use Discord to provide support for our users, collaborate with contributors and discuss development with our patrons.*"